.changeset/metal-trainers-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"permissionless": patch
+---
+
+fix: domain typing
```

Comment:
Can we change to `Fixed domain not typed as TypedDataDomain`

Reply:
Gotcha first. Open source commit, will be more specific in the future.